Source/appfat.cpp

```diff
@@ -15,9 +15,9 @@ namespace devilution {
 namespace {
 
 /** Set to true when a fatal error is encountered and the application should shut down. */
-bool terminating = false;
+bool Terminating = false;

```

Comment:
`GlobalVariableCase,      value: aNy_CasE`, so why did this change?

Reply:
I applied the rules to `appfat.cpp` without issues, only when getting to the later files did I need to permit "aNy_CasE" for global names. Once everything else has been applied I intent to switch it back to `CamelCase` and apply it globally to the project.

Reply:
Maybe also add some prefix to the globals?